### DISCUSSION TITLE
Question about components

### DISCUSSION THREAD

**Discussion by StructureIsKey** [ozel/DIY_particle_detector#43] [Q&A] [answered]
Hello,

First of all: thank you for this amazing project and very instructive and detailed documentation.

I was wondering how 'strict' the requirement for the components is, such as: do all the resistors (beside the SMD) need to have a max power of 0.6W? Would 0.25W work fine?

Thanks a lot!

**Comment by ozel** [comment, marked as answer]
Hello,

glad you enjoy the project!

The resistor power specification is not relevant at all. I chose part numbers for larger 0.6 W resistors as they are a bit easier to solder for beginners and may have slightly better availability. But you should stick to metal film resistors in general.

The circuit draws very little current and dissipates less than 50 Milliwatt overall.
See https://github.com/ozel/DIY_particle_detector/wiki/Batteries

**Reply by aeonSolutions** [reply]
I'm thinking adding a ESP32 S3,

quick question: can i replace both resistors and capacitors for SMD 0402\0603 components ? 

🙏

**Reply by ozel** [reply]
yes, of course
